sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileIO.java

```diff
@@ -305,19 +306,21 @@ public MatchAll continuously(
 
     @Override
     public PCollection<MatchResult.Metadata> expand(PCollection<String> input) {
+      PCollection<MatchResult.Metadata> res;
       if (getConfiguration().getWatchInterval() == null) {
-        return input.apply(
+        res = input.apply(
```

Comment:
Don't use abbreviations like res when result is clearer and not much longer.

Reply:
Both seem used quite a lot inside Beam (granted, "result" is used more); I don't think a convincing argument can be made that "res" is unclear :) But ok, I'll change before merging.